environment.go

```diff
@@ -215,8 +215,22 @@ func (e *envState) initClientConfig() (err error) {
 }
 
 type bbProject struct {
-	TicketCreateProject  string   `mapstructure:"ticket_create_project" bson:"ticket_create_project"`
-	TicketSearchProjects []string `mapstructure:"ticket_search_projects" bson:"ticket_search_projects"`
+	TicketCreateProject  string   `mapstructure:"ticket_create_project"`
+	TicketSearchProjects []string `mapstructure:"ticket_search_projects"`
+
+	AlternativeEndpointURL struct {
```

Comment:
Can you define these structs outside the bbProject struct? Also please make them private

Reply:
make this public and delete the copy. I'm fine with embedded structs